User Data Netplan (#1705)
spectrocloud/librarium: docs/docs-content/clusters/edge/edge-configuration/cloud-init.md

                  insecure_skip_verify: true
```

#### Configure Network With Netplan

> You can use the `initramfs` stage and [Netplan](https://netplan.io) to configure network settings before the network initialization. Netplan is a tool that enables you to specify network configurations on Linux systems. Please note that this approach is available for Ubuntu systems only. Refer to the [Netplan Documentation](https://netplan.readthedocs.io/en/stable/) and [Netplan How-to Guides](https://netplan.readthedocs.io/en/stable/examples/) for more information.

Reviewer: ```suggestion
You can use the `initramfs` stage and [Netplan](https://netplan.io) to configure network settings before the network initialization. Netplan is a tool that enables you to specify network configurations on Linux systems. Note that this approach is available for Linux systems with Netplan installed. Refer to the [Netplan Documentation](https://netplan.readthedocs.io/en/stable/)  for install guidance and the [Netplan How-to Guides](https://netplan.readthedocs.io/en/stable/examples/) for more information.
```

Per our style guide, we don't use please.
